packages/nimble-components/src/rich-text/mention-listbox/tests/mention-listbox.spec.ts

```diff
@@ -44,62 +44,82 @@ describe('RichTextMentionListbox', () => {
         );
     }
 
+    let element: RichTextMentionListbox;
+    let anchor: HTMLElement;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
     async function waitForSelectionUpdateAsync(): Promise<void> {
         await waitForUpdatesAsync();
         await waitAnimationFrame(); // necessary because scrolling is queued with requestAnimationFrame
     }
 
-    async function showAndWaitForOpen(
-        listbox: RichTextMentionListbox,
-        anchor: HTMLElement
-    ): Promise<void> {
-        const regionLoadedListener = createEventListener(listbox, 'loaded');
-        listbox.show({
+    async function showAndWaitForOpen(filter = ''): Promise<void> {
+        const regionLoadedListener = createEventListener(element, 'loaded');
+        element.show({
             anchorNode: anchor,
-            filter: ''
+            filter
         });
         await regionLoadedListener.promise;
     }
 
```

Comment:
Think it'd be worthwhile to include a test that the No Items Found item is not shown if the set of items updates to include an item that matches the filter? i.e. that we are correctly observing the child list options